src/resolvers/admin.ts

```diff
@@ -23,4 +24,10 @@ export class AdminResolver {
     await killSwitch.save()
     return { isKilled }
   }
+
+  @Mutation(() => GeneratedSignature)
+  async signAppActionCommitment(@Arg('message', () => String) message: string) {
+    const signature = sr25519Sign(message, config.appKeypair)
+    return { signature: Buffer.from(signature).toString('hex') }
```

Comment:
I think you could use `u8aToHex` from `@polkadot/util` instead of using `Buffer` directly